docs/sections/external_links.rst

```diff
@@ -0,0 +1,8 @@
+.. _anaconda: https://anaconda.org/maddenp/condev  
+.. _black: https://github.com/black-software-Com/Black-Tool
```

Comment:
I don't think this is the right `black`. I think this link would be better: https://black.readthedocs.io/en/stable/

Reply:
Yes, I copied and pasted the wrong link. Fixing that now, thanks for the catch!